Jakefile.js

```diff
@@ -1031,3 +1159,85 @@ task("lint-server", ["build-rules"], function() {
         lintWatchFile(lintTargets[i]);
     }
 });
+
+function ProgressBar(title) {

```

Comment:
That one is part of a mocha reporter. This Is used by a task that runs mocha using the TAP reporter so there is no progress. It's just something small I threw together so you know something is still happening.